apps/block/google_news_result.ts

```diff
@@ -67,7 +67,7 @@ export class GoogleNewsResult extends SearchResultToBlock {
     );
     this.valid = true;
     this._canRetry = true;
-    this.url = href;
+    this.url = href !== null ? href : "";
```

Comment:
@vibh1103 

If href is null, the url will be empty and should not be blocked.
Please refer to the previous titleElement code(L50).

Reply:
Ok

Reply:
Done